src/C++/SessionFactory.cpp

```diff
@@ -201,7 +201,15 @@ Session* SessionFactory::create( const SessionID& sessionID,
   if ( settings.has( REFRESH_ON_LOGON ) )
     pSession->setRefreshOnLogon( settings.getBool( REFRESH_ON_LOGON ) );
   if ( settings.has( MILLISECONDS_IN_TIMESTAMP ) )
-    pSession->setTimestampPrecision(3);
+  {
```

Comment:
We have a helper for this, this should just become

```
if ( settings.has( MILLISECONDS_IN_TIMESTAMP ) )
    pSession->setMillisecondsInTimeStamp( settings.getBool( MILLISECONDS_IN_TIMESTAMP ) );
```

Reply:
Thank you. It's an elegant way. I've pushed new commit.